### ISSUE TITLE
Improve accessibility by adding [lang] attribute to html element 

### ISSUE THREAD

**Issue by fanyuex2** [CMU-313/Teedy#9]
The google lighthouse showed the current accessibility score is 86 with a warning "<html> element does not have a [lang] attribute" for INTERNATIONALIZATION AND LOCALIZATION section. By specifying the language, the webpage can better locate the target audience.

**Comment by fanyuex2** [comment]
I would like to work on this please!
